server/controllers/PropertyController.js

```diff
@@ -15,18 +16,35 @@ cloudinary.config({
 });
 
 const { response } = Respond;
+
+// db helpers  funcs
+const { queryAll, find } = DbHelper;
+
 class PropertyController {
   // view all properties
-  static viewAllProperties(req, res) {
-    return response(res, 200, properties);
+  static async viewAllProperties(req, res) {
+    const { error, response: result } = await queryAll('properties');
+    if (error) {
+      return response(res, 404, error, true);
```

Comment:
I don't this is a 404 error. It's not that a resource doesn't exist, but rather the server crashed for a number of reasons. I would rather you return an error 500.